test/blackbox-tests/test-cases/describe/aliases.t/run.t

```diff
@@ -7,6 +7,9 @@ In an empty dune project, the following aliases are available.
   $ dune show aliases
   all
   default
+    - This alias corresponds to the default argument for dune build. More
+      information on
+      https://dune.readthedocs.io/en/latest/reference/aliases/default.html
   fmt
```

Comment:
I wonder, maybe it would make sense to add these default descriptions to all aliases. Or to none. But at least consistently.

Reply:
I was thinking for a while now about documentation for built-in aliases.

I am going to drop it completely in this PR and will re-implement UX for `dune show aliases` in following PRs to link to built-in doc in the very top of output.

Reply:
Removed for now in 02f3c1045

Will re-iterate in future PRs